pkg/grpc/proto/grpcservice.proto

```diff
@@ -3,21 +3,25 @@ syntax = "proto3";
 package grpc;
 option go_package = "github.com/stasBigunenko/monorepa/pkg/grpc/proto";
 
+import "google/protobuf/empty.proto";
 
-service GrpcService {
-  rpc GetItems(Username) returns (Items) {}
-}
 
-message Items {
-  repeated Obj items = 1;
+service UserService {
+  rpc GetUser(UserID) returns (Obj) {}
+  rpc CreateUser(Username) returns (Obj) {}
+  rpc UpdateUser(Obj) returns (Obj) {}
+  rpc DeleteUser(UserID) returns (google.protobuf.Empty) {}
 }
 
 message Obj {
   string id = 1;
```

Comment:
Obj is a meaningless name for a struct. Call it User. GetUser should return User, not Obj.

Reply:
done